File: DSA/Manav-Rajput/Day12_question_1.java
public class Fibonacci {

    // Function to compute the n-th Fibonacci number
    public static int fibonacci(int n) {
        // Base cases
        if (n <= 0) {
            return 0;
        } else if (n == 1) {
            return 1;
        } else {
            // Recursive call
            return fibonacci(n - 1) + fibonacci(n - 2);
        }
    }

    // Main method to test the function
    public static void main(String[] args) {
        int n = 5;  // You can change this value to test other inputs
        int result = fibonacci(n);
        System.out.println("Fibonacci number at position " + n + " is: " + result);
    }
}

File: DSA/Ansh-Gulati/Day-12/Day-12_Question-1.cpp
#include <iostream>

using namespace std;

int fibonacci(int n)
{
    if (n == 0)
    {
        return 0;
    }
    else if (n == 1)
    {
        return 1;
    }
    else
    {
        return fibonacci(n - 1) + fibonacci(n - 2);
    }
}

int main()
{
    int n;
    cout << "Enter a positive number: ";
    cin >> n;
    if (n < 0)
    {
        cout << "Error: enter a positive number." << endl;
        return 1;
    }
    int result = fibonacci(n);
    cout << "Fibonacci number at position " << n << " is: " << result << endl;
    return 0;
}
